io-pay-portal-fe/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "type-check": "tsc",
     "generate": "npm-run-all generate:*",
-    "generate:api": "rimraf generated && mkdir -p generated && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-functions-checkout/v1.1.2/openapi/index.yaml --no-strict --out-dir ./generated --request-types --response-decoders --client",
+    "generate:api": "rimraf generated && mkdir -p generated && gen-api-models --api-spec REMOVE_THIS_api/openapi.yml --no-strict --out-dir ./generated --request-types --response-decoders --client",
```

Comment:
you can use https://github.com/pagopa/io-pagopa-proxy/blob/main/api-spec/api-for-checkout.yaml

Reply:
wait, let's create a tag first 😁

Reply:
https://raw.githubusercontent.com/pagopa/io-pagopa-proxy/v1.1.2/api-spec/api-for-checkout.yaml @giovanniberti